### DISCUSSION TITLE
LEO Satellite Elevation Angle - Skyplot (receiver?)

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#168] [Q&A] [answered]
Hello,

I want to plot the elevation angle of a LEO Satellite and i saw in the documentation the script "GNSSresiduals2Skyplot" that mentions elevation angle at receiver - Would this receiver also be the LEO satellite receiver (similar to a fixed station receiver)? I am asking the question to be sure that i am will be using the correct script to plot the LEO satellite elevation angle? Or did i understand this not correctly?  If this is not correct, any good advice on how to go about it then?

Txs for the feedback and the continuous help.

Cheers
Jean

**Comment by tmayerguerr** [comment, marked as answer]
Hi Jean,

[GnssResiduals2Skyplot](https://groops-devs.github.io/groops/html/GnssResiduals2Skyplot.html) should work for receivers mounted at LEO satellites as well. The elevation at each time is relative to the changing orientation (horizon) of the receiver.

regards
Torsten

**Reply by Bemtgenj** [reply]
Hi Torsten, 
Txs for the feedback and clarification. We will try this script.

Cheers
Jean

**Comment by Bemtgenj** [comment]
Good  Morning Torsten,

I have followed your advice and used first GnssResiduals2Skyplot to create a griddedData File (Longitude, Latitude, Height, Unit Areas, data0, data1) After that i used the plotMap script with data0 as input. In this case data0 is the fifth column (and not data0  being a "first column")?  If this is correct then data0 represents either the C or L residuals (which ever i listed first in the GNSS type between those two). The Skyplot based on data0 i got after changing some of the R and I values. What is not clear to me then is how i can print the elevation angle from this same griddedData file?

TXs for the help and feedback

Cheers
Jean

**Comment by tmayerguerr** [comment]
Hi Jean,
The gridded data file contains Longitude=azimuth, Latitude=elevation and you can use [PlotMap](https://groops-devs.github.io/groops/html/PlotMap.html) with [projection](https://groops-devs.github.io/groops/html/plotMapProjectionType.html)->Skyplot to visualize the result.
regards
Torsten

**Comment by Bemtgenj** [comment]
Hi Torsten

Thank you very much for the feedback - now that i look at it again that makes a lot of sense now.

Cheers
Jean

**Comment by gnssLa** [comment]
@tmayerguerr,

A similar issue:  I wanted a figure that displays skyplots in a row. I was wondering if you could kindly provide me with some advice or a sample script on how to generate skyplots in a similar fashion. 

Using PlotMap[](https://groops-devs.github.io/groops/html/GnssResiduals2Skyplot.html): 

1. When I attempt to generate skyplots with multiple layers, the resulting plots overlap each other.
2. I am unable to locate the setting that would allow me to align multiple plots using a single color bar.

Here is an example of what I need: 

**Skyplots in a row:**
![exampleSkyPlot](https://github.com/groops-devs/groops/assets/109293325/bdfdf7d5-308e-4c57-868c-227c31e91f74)

similar to what is [here.](https://groops-devs.github.io/groops/html/fileFormat_gnssAntennaDefinition.html).

I appreciate your help in advance.

Regards,

Duarte.

**Reply by tmayerguerr** [reply]
Hi Duarte,

In fact, GROOPS is not able to create several plots next to each other in one file. However, you can create several plots and then join them with any graphics programme.

Just as a hint: If you switch off removeFiles in options, you will get a folder with all data and a script for GMT in the output directory, which you can then modify as you like.

regards
Torsten

**Comment by gnssLa** [comment]
Dear @tmayerguerr,

Thanks for your reply. I will do as advised. 

Regards,

Duarte.
